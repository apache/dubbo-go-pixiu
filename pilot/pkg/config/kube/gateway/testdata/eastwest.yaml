apiVersion: gateway.networking.k8s.io/v1alpha2
kind: Gateway
metadata:
  name: eastwestgateway
  namespace: dubbo-system
  labels:
    topology.istio.io/network: "network-1"
spec:
  gatewayClassName: istio
  listeners:
  - name: istiod-grpc
    port: 15012
    protocol: TLS
    tls:
      mode: Passthrough
  - name: istiod-webhook
    port: 15017
    protocol: TLS
    tls:
      mode: Passthrough
  - name: cross-network
    hostname: "*.local"
    port: 15443
    protocol: TLS
    tls:
      mode: Passthrough
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: TLSRoute
metadata:
  name: eastwestgateway-grpc
  namespace: dubbo-system
spec:
  parentRefs:
  - name: eastwestgateway
    kind: Gateway
    sectionName: istiod-grpc
  rules:
  - backendRefs:
    - name: istiod
      port: 15012
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: TLSRoute
metadata:
  name: eastwestgateway-webhook
  namespace: dubbo-system
spec:
  parentRefs:
  - name: eastwestgateway
    kind: Gateway
    sectionName: istiod-webhook
  rules:
  - backendRefs:
    - name: istiod
      port: 15017
