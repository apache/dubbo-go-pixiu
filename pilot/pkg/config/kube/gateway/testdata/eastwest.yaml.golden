apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  annotations:
    internal.istio.io/gateway-service: eastwestgateway.istio-system.svc.domain.suffix
    internal.istio.io/parent: Gateway/eastwestgateway/istiod-grpc.istio-system
  creationTimestamp: null
  name: eastwestgateway-istio-autogenerated-k8s-gateway-istiod-grpc
  namespace: istio-system
spec:
  servers:
  - hosts:
    - istio-system/*
    port:
      name: default
      number: 15012
      protocol: TLS
    tls: {}
---
apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  annotations:
    internal.istio.io/gateway-service: eastwestgateway.istio-system.svc.domain.suffix
    internal.istio.io/parent: Gateway/eastwestgateway/istiod-webhook.istio-system
  creationTimestamp: null
  name: eastwestgateway-istio-autogenerated-k8s-gateway-istiod-webhook
  namespace: istio-system
spec:
  servers:
  - hosts:
    - istio-system/*
    port:
      name: default
      number: 15017
      protocol: TLS
    tls: {}
---
apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  annotations:
    internal.istio.io/gateway-service: eastwestgateway.istio-system.svc.domain.suffix
    internal.istio.io/parent: Gateway/eastwestgateway/cross-network.istio-system
  creationTimestamp: null
  name: eastwestgateway-istio-autogenerated-k8s-gateway-cross-network
  namespace: istio-system
spec:
  servers:
  - hosts:
    - istio-system/*.local
    port:
      name: default
      number: 15443
      protocol: TLS
    tls:
      mode: AUTO_PASSTHROUGH
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/parent: TLSRoute/eastwestgateway-grpc.istio-system
    internal.istio.io/route-semantics: gateway
  creationTimestamp: null
  name: eastwestgateway-grpc-tls-0-istio-autogenerated-k8s-gateway
  namespace: istio-system
spec:
  gateways:
  - istio-system/eastwestgateway-istio-autogenerated-k8s-gateway-istiod-grpc
  hosts:
  - '*'
  tls:
  - match:
    - sniHosts:
      - '*'
    route:
    - destination:
        host: istiod.istio-system.svc.domain.suffix
        port:
          number: 15012
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/parent: TLSRoute/eastwestgateway-webhook.istio-system
    internal.istio.io/route-semantics: gateway
  creationTimestamp: null
  name: eastwestgateway-webhook-tls-0-istio-autogenerated-k8s-gateway
  namespace: istio-system
spec:
  gateways:
  - istio-system/eastwestgateway-istio-autogenerated-k8s-gateway-istiod-webhook
  hosts:
  - '*'
  tls:
  - match:
    - sniHosts:
      - '*'
    route:
    - destination:
        host: istiod.istio-system.svc.domain.suffix
        port:
          number: 15017
---
